### DISCUSSION TITLE
War effort

### DISCUSSION THREAD

**Discussion by ZedNox** [chromiecraft/chromiecraft#4050] [Suggestions]
Are we going to have 1month of war effort event? suggest to lower the things for war effort or there's gonna be half server with scarab lord wich is meaningless at that point.

**Comment by BeefSteaker** [comment]
Progress is already decent, and will speed up once the big guilds are done with scarab lord

**Reply by ZedNox** [reply]
yeah sure let's make whole raid scarab lord. it's gonna be good.
